projects/etcd.yml

```
@@ -0,0 +1,6 @@
+name: etcd
+url: https://github.com/etcd-io/etcd
+modules:
+mixins:
+  - import: github.com/etcd-io/etcd/Documentation/etcd-mixin
```

```suggestion
  - import: github.com/etcd-io/etcd/documentation/etcd-mixin
```

Any reason for this to be uppercase?